lib/src/extensions/widget_extension.dart

```diff
@@ -727,8 +727,10 @@ extension StyledWidget on Widget {
               transformHitTests: transformHitTests,
             );
 
-  Widget scale(
-    double scale, {
+  Widget scale({
+    double all,
```

Comment:
Consider changing `all` to `proportional`? Worth the longer word to make it more descriptive?

Reply:
I think the fine, but short word may help keep the code concise, just like `EdgeInsets.all`?

Reply:
Okey, i will accept your changes. Thanks for contributing to the project!